libraries/freertos_plus/standard/pkcs11/src/iot_pkcs11.c

```diff
@@ -1,3 +1,28 @@
+/*
+ * Amazon FreeRTOS PKCS#11 V1.0.0
```

Comment:
Is this supposed to be V2.0.0?

Reply:
`PKCS #11`

Reply:
libraries/freertos_plus/standard/pkcs11 is 1.0.0 and 

libraries/abstractions/pkcs11 is 2.0.0